### PR TITLE
`station logs [<module>]`. Closes #22. Closes #19

### LINKED ISSUE #19: `station --help`

### LINKED ISSUE #22: `station logs [<module>]`

### PR DESCRIPTION
**Pull request by juliangruber**
Closes #22.
Closes #19.

In order to get all logs, I decided to also write all logs to a combined `all.log`, instead of merging and sorting lines from multiple files while reading, which is significantly more complex.

Since argument parsing was becoming complex enough, I introduced `yargs`. I decided for it because I know it well and it is convenient. I decided against `minimist` since it doesn't provide `--help` and other nice features. I decided against another framework because I didn't know it and didn't feel the need to replace `yargs`.

### CODE REVIEW

**Comment by bajtos** [comment]
> Since argument parsing was becoming complex enough, I introduced `yargs`. I decided for it because I know it well and it is convenient. I decided against `minimist` since it doesn't provide `--help` and other nice features. I decided against another framework because I didn't know it and didn't feel the need to replace `yargs`.

Cool! I was worried that `yargs` and `minimist` may be abandoned but was pleasantly surprised that at least `yargs` is still actively maintained 👌🏻

File: commands/logs.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  }
+}
+
+const followLogs = path => {
+  const tail = new Tail(path, { nLines: 10 })
+  tail.on('line', line => console.log(line))
+}
+
+const getLogs = async path => {
+  process.stdout.end(await fs.readFile(path))
```
Comment (bajtos) [inline commands/logs.js:22]: I find it surprising that `getLogs` has a significant side-effect of closing the `stdout` stream.

I am proposing two changes:
1. Rename `getLogs` to something like `printLogs`
2. Can we write the logs without closing stdout?

**Review by bajtos — APPROVED** [review]
👍🏻

File: test/test.js [hunk 2]
```diff
@@ -102,6 +107,45 @@ test('Metrics', async t => {
...
+    for (const flag of ['-f', '--follow']) {
+      const XDG_STATE_HOME = join(tmpdir(), randomUUID())
+      await fs.mkdir(getPaths(XDG_STATE_HOME).moduleLogs, { recursive: true })
+      const ps = execa(station, ['logs', flag], { env: { XDG_STATE_HOME } })
+      await Promise.all([
+        once(ps.stdout, 'data'),
+        fs.writeFile(getPaths(XDG_STATE_HOME).allLogs, '[date] beep boop\n')
+      ])
+      ps.kill()
+    }
```
Comment (bajtos) [inline test/test.js:145]: This test does not seem to assert anything. If that's intentional, then please add a comment to explain how this test works.

File: commands/logs.js (hunk 1 above)
Comment (juliangruber) [reply commands/logs.js]: Good catch! I was over-eager with the `.end`

**Pull request merged**